build.bat

```diff
@@ -507,8 +506,9 @@ REM BUILD function: build various targets
 		rem call:BUILDTYPE
 
 		set CMAKE_ARGS=-DDRIVER_BASE_NAME=%DRIVER_BASE_NAME%
-		REM no explicit x86 generator and is the default (MSVC2017 only?).
-		set CMAKE_ARGS=!CMAKE_ARGS! -DCMAKE_GENERATOR_PLATFORM=%TARCH:x86=%
+		REM CMAKE_GENERATOR_TOOLSET (or cmake -T v142) to specify a toolset
+		REM cmake's architecture specs for MSVC are x64 and Win32
+		set CMAKE_ARGS=!CMAKE_ARGS! -DCMAKE_GENERATOR_PLATFORM=!TARCH:x86=Win32!
```

Comment:
```suggestion
		set CMAKE_ARGS=!CMAKE_ARGS! -G "Visual Studio 16 2019" -DCMAKE_GENERATOR_PLATFORM=!TARCH:x86=Win32!
```

...so that it keeps working when Visual Studio 2021 is added to the build machine.